start-from-zero/Javascript/src/data/storeData.js

```diff
@@ -1,45 +1,50 @@
-windowObjectWrap.FormData = function(){
-    btn.addEventListener('click',function(){
-        function User(username,date,radio,textarea,select){
-        this.username = username;
-        this.date = date;
-        this.radio = radio;
-        this.textarea = textarea.forEach(element=>{
-            this[element.name] = element.value;
-        });
-        this.select = select.forEach(element=>{
-            this[element.name] = element.value;
-        })
-        }
-
-        const username = document.getElementById('username-id').value;
-        const calendar = document.getElementById('calendar-id').value;
-        const checkbox = document.querySelectorAll('.input-checkbox');
-        const textarea = document.querySelectorAll('textarea');
-        const legend = document.querySelectorAll('legend');
-        const select = document.querySelectorAll('select');
-    
-        const radioArr = [];
-        checkbox.forEach(element => {
-            if(element.checked === true){
-                radioArr.push(element.value);
+(function(){
+    windowObjectWrap.FormData = function(){
+        btn.addEventListener('click',function(){
+            function User(username,date,radio,textarea,select,id){
+            this.username = username;
+            this.date = date;
+            this.radio = radio;
+            this.textarea = textarea.forEach(element=>{
+                this[element.name] = element.value;
+            });
+            this.select = select.forEach(element=>{
```

Comment:
Same question as above. What value does `this.select` hold when the iteration finishes? Is it what you expect it to be? Also, what does the callback itself do?

Reply:
It populates the object with the key and the value of the select element.